src/view/com/profile/ProfileCard.tsx

```diff
@@ -25,6 +25,7 @@ export const ProfileCard = observer(
     followers,
     overrideModeration,
     renderButton,
+    asAnchor = true,
```

Comment:
Did you mean to include this? Seems like a fine idea, just double checking

Reply:
`asAnchor` was the default for `ProfileCard` so that is why. I had to remove it otherwise when you clicked on the dropdown it went to another page.